cylc/flow/cfgspec/globalcfg.py

```diff
@@ -763,22 +764,20 @@
                 systems so for safety there is an upper limit on the number
                 of job submissions which can be batched together.
             ''')
-            with Conf('selection'):
-                Conf(
-                    'method', VDR.V_STRING, default='random',
-                    options=['random', 'definition order'],
-                    desc='''
-                        Host selection method for the platform. Available
-                        options:
-
-                        - random: Suitable for an identical pool of hosts.
-                        - definition order: Take the first host in the list
-                          unless that host has been unreachable. In many cases
-                          this is likely to cause load imbalances, but might
-                          be appropriate if your hosts were
-                          ``main, backup, failsafe``.
-                    '''
-                )
+            Conf('host selection method', VDR.V_STRING, default='random',
+                 options=['random', 'definition order'],
+                 desc='''
+                Method for choosing the job host from the platform.
```

Comment:
```suggestion
                Method for choosing the job host from the platform.

```

In RST you need a blank line to separate two paragraphs.